src/ucs/datastruct/bitmap.h

```diff
@@ -370,7 +370,8 @@ typedef uint64_t ucs_bitmap_word_t;
          _bit_index < _UCS_BITMAP_NUM_WORDS(_bitmap) * UCS_BITMAP_BITS_IN_WORD; \
          _bit_index = _UCS_BITMAP_WORD_INDEX0(_bit_index) + \
                   ucs_ffs64_safe(_UCS_BITMAP_WORD_BY_BIT((_bitmap), (_bit_index)) & \
-                                 _UCS_BITMAP_GET_NEXT_BIT(_bit_index)))
+                                 _UCS_BITMAP_GET_NEXT_BIT(_bit_index))) \
+        if (UCS_BITMAP_GET(_bitmap, _bit_index)) 
```

Comment:
can we implement something like "ucs_bitmap_ffs(start_index, " helper function, which will be used to get next _bit_index?

Reply:
Making it a function requires quite dirty pointer manipulations, as `_bitmap` can't be passed with its true type in this context, but only as `void *`. Are you sure we want that?